ui/pages/create-snap-account/create-snap-account.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import CreateSnapAccount from './create-snap-account';
+
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
why is this needed?

Reply:
Most of the storybook files are plain js which have less strict/different lint rules. When this line was not present the linter would fail. I believe this is the standard pattern with storybook files so I left the implementation as is and added this lint ignore. 
<img width="968" alt="Screenshot 2023-09-28 at 10 45 53 AM" src="https://github.com/MetaMask/metamask-extension/assets/22918444/359fa751-cae6-449f-8afe-a2e1503f2549">